Deploy/bash/geteob.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Set to the name of the service you're testing
+SANDBOX=http://sandbox.ab2d.cms.gov/api/v1/fhir
```

Comment:
Since this is a shell script to run from command line, would it make more sense to pass these values via command line parameters instead? `./geteob --url URL --clientId ... --clientPass ...`?

Reply:
There are actually a lot of things we could do (there is very little error checking). I mostly used this as the verifiable way to make sure the commands I put in my Curl tutorial https://confluence.cms.gov/display/AB2D/Curl+Tutorial was correct and as a means to support our development. I think the documentation in the tutorial is sufficient that the script doesn't really need to be included on the web site and it can be useful for us. Maybe abstracting the URL and the base64 encoded client id and password are sufficient. Actually, probably also the contract number as an optional parameter.